CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ project(ohNet LANGUAGES CXX C)
 
 set(platform Posix)
 
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
The currently preferred way is to use Conan with find_package generator and use `find_packages` and `target_link_libraries` individually for each imported lib.

See for reference: https://github.com/PacktPublishing/Software-Architecture-with-Cpp/blob/master/Chapter08/customer/CMakeLists.txt#L21